src/paragraph-break.js

```diff
@@ -0,0 +1,51 @@
+/**
+📃
+@file paragraph-break
+@summary Adds paragraph breaks to the dialogue parser
+@license WTFPL (do WTF you want)
+@version 1.0.0
+@requires Bitsy Version: 5.0, 5.1
+@author Sean S. LeBlanc, David Mowatt
+
+@description
+Adds a (p) tag to the dialogue parser that forces the following text to the
+start of a fresh dialogue screen, eliminating the need to spend hours testing
+line lengths or adding multiple line breaks that then have to be reviewed
+when you make edits or change the font size.
+
+Usage: (p)
+       
+Example: I am a cat(p)and my dialogue contains multitudes
+
+HOW TO USE:
+  1. Copy-paste this script into a new script tag after the Bitsy source code.
+     It should appear *before* any other mods that handle loading your game
+     data so it executes *after* them (last-in first-out).
+
+NOTE: This uses parentheses "()" instead of curly braces "{}" around function
+      calls because the Bitsy editor's fancy dialog window strips unrecognized
+      curly-brace functions from dialog text. To keep from losing data, write
+      these function calls with parentheses like the examples above.
+
+      For full editor integration, you'd *probably* also need to paste this
+      code at the end of the editor's `bitsy.js` file. Untested.
+*/
+'use strict';
+import bitsy from "bitsy";
+import {
+	addDialogTag,
+} from "./helpers/kitsy-script-toolkit";
+
+
+//The actual function; uses the existing AddLinebreak function to avoid scope issues.
```

Comment:
this comment isn't super helpful; we both know what it's talking about since we just dealt with it, but it's gonna mean nothing to people seeing it for the first time, and probably also us a couple months from now

it'd be better to describe what it's doing and/or how it's doing it vs referencing a way that didn't work